### ISSUE TITLE
RGB beacons

### ISSUE THREAD

**Issue by SwissalpS** [OgelGames/beacon#15]
Idea that crossed my mind: craft with red, green and blue dye and allow colour change via digiline.
only the ten available colours, not true rgb.

**Comment by OgelGames** [comment]
Hmm... Not sure how practical it would be with regards to lag production from changing beams, but it's a interesting idea...

**Comment by SwissalpS** [comment]
yes, may want to reduce the change to only when activated. It won't stop some from using it continuously.

**Comment by S-S-X** [comment]
New color can be simply used during normal beam forming which I think is not available as digiline command.
So allow setting color that will be used but do not actually change color / update beam but instead store color and use it in next beam forming. Probably also want to preserve color when beacon is digged, this adds metadata to stack and infotext or inventory image should reflect stack rgb setting somehow.

Formspec field would probably be better for this instead of setting color with digiline command, there one can also easily deactivate/activate and therefore change beam color.

**Comment by SwissalpS** [comment]
> New color can be simply used during normal beam forming which I think is not available as digiline command.
there is activate command now, have been using it for the newest beam factories.

After digging colour can be dropped. It can be set again after place via luac or possibly also formspec.
